#! /bin/sh /usr/share/dpatch/dpatch-run
## 06default_value_escapetime.dpatch by  <hesso@pool.math.tu-berlin.de>
##
## DP: Increase the default 'escapetime' setting to accommodate slow lines.

@DPATCH@
diff -Naur nvi-1.81.6.orig/common/options.c nvi-1.81.6/common/options.c
--- nvi-1.81.6.orig/common/options.c	2007-11-18 17:41:42.000000000 +0100
+++ nvi-1.81.6/common/options.c	2008-05-01 18:06:18.000000000 +0200
@@ -356,7 +356,7 @@
 	(void)SPRINTF(b2, SIZE(b2),
 	    L("directory=%s"), (s = getenv("TMPDIR")) == NULL ? _PATH_TMP : s);
 	OI(O_TMP_DIRECTORY, b2);
-	OI(O_ESCAPETIME, L("escapetime=1"));
+	OI(O_ESCAPETIME, L("escapetime=3"));
 	OI(O_KEYTIME, L("keytime=6"));
 	OI(O_MATCHTIME, L("matchtime=7"));
 	(void)SPRINTF(b2, SIZE(b2), L("msgcat=%s"), _PATH_MSGCAT);
